Add details to the README. (#64)
Financial-Times/create-origami-component: README.md

# create-origami-component

A command line tool for creating new Origami components.

## Usage

```shell
npm init origami-component
```

## Contact

If you have any questions or comments about this project, or need help using it, please either [raise an issue](https://github.com/Financial-Times/create-origami-component/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).

----

## License

> Copyright (c) 2016 Financial Times Ltd. All rights reserved.

Reviewer: ```suggestion
Copyright (c) 2020 Financial Times Ltd. All rights reserved.
```

> Copyright (c) 2016 Financial Times Ltd. All rights reserved.

Author: ```suggestion
Copyright (c) 2020 Financial Times Ltd. All rights reserved.
```

> If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/o-subs-card/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).		 If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/o-subs-card/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).

Reviewer: ```suggestion
If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/o-subs-card/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).		 If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/.../issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).
```

Reviewer: the link had o-subs-card, I replaced with ... as I have no idea what to replace with 🙈

Author: sorry I spotted and already resolved it, thanks for flagging!

> If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/o-subs-card/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).		 If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/o-subs-card/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).

Author: Copy and paste got me 
```suggestion
If you have any questions or comments about this project, or need help using it, please either [raise an issue](https://github.com/Financial-Times/create-origami-component/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).
```

> Copyright (c) 2016 Financial Times Ltd. All rights reserved.

Reviewer: that license year would be wrong, it should be the year they run the command?

Author: copy paste got me again
